cve.py

```diff
@@ -5,16 +5,23 @@ def __init__(self, dict : {}):
         """
 
         """
-        self.ID = dict['cve']['CVE_data_meta']['ID']
+        self.identifier = dict['cve']['CVE_data_meta']['ID']
         self.ASSIGNER = dict['cve']['CVE_data_meta']['ASSIGNER']
         self.description = dict['cve']['problemtype']['problemtype_data'][0]['description'][0]['value']
+        self.severity = dict['impact']
```

Comment:
This logic shouldn't be in a simple data structure class. should get the severity without knowing about v2 or v3